python/taichi/code_format.py

```diff
@@ -103,30 +108,31 @@ def find_diff_or_empty(s):
         if fn.find('docs/build/') != -1:
             continue
         if re.match(r'.*examples\/[a-z_]+\d\d+\.py$', fn):
-            print(f'Skipping example file {fn}...')
+            print(f'Skipping example file "{fn}"...')
             continue
-        if fn.endswith('.py'):
-            print('Formatting "{}"'.format(fn))
-            FormatFile(fn,
-                       in_place=True,
-                       style_config=os.path.join(repo_dir, 'misc',
-                                                 '.style.yapf'))
-        elif clang_format_bin and has_suffix(fn, ['cpp', 'h', 'cu', 'cuh']):
-            print('Formatting "{}"'.format(fn))
-            os.system('{} -i -style=file {}'.format(clang_format_bin, fn))
-        elif has_suffix(fn, ['txt', 'md', 'rst', 'cfg', 'll', 'ptx']):
-            print('Formatting "{}"'.format(fn))
-            format_plain_text(fn)
-        elif has_suffix(fn, [
-                'pyc', 'png', 'jpg', 'bmp', 'gif', 'gitignore', 'whl', 'mp4',
-                'html'
-        ]):
-            pass
-        else:
-            print(f'Skipping {fn}...')
+        if not do_format(fn):
+            print(f'Skipping "{fn}"...')
 
     print('Formatting done!')
 
 
+def do_format(fn):
+    clang_format_bin = find_clang_format_bin()
+    if fn.endswith('.py'):
+        print('Formatting "{}"'.format(fn))
+        FormatFile(fn,
+                   in_place=True,
+                   style_config=os.path.join(repo_dir, 'misc', '.style.yapf'))
+    elif clang_format_bin and has_suffix(fn, ['cpp', 'h', 'cu', 'cuh']):
+        print('Formatting "{}"'.format(fn))
+        os.system('{} -i -style=file {}'.format(clang_format_bin, fn))
+    elif has_suffix(fn, ['txt', 'md', 'rst', 'cfg', 'll', 'ptx']):
+        print('Formatting "{}"'.format(fn))
+    else:
+        return False
+    format_plain_text(fn)
+    return True
```

Comment:
```suggestion
    if fn.endswith('.py'):
        print('Formatting "{}"'.format(fn))
        FormatFile(fn,
                   in_place=True,
                   style_config=os.path.join(repo_dir, 'misc', '.style.yapf'))
        return True
    elif clang_format_bin and has_suffix(fn, ['cpp', 'h', 'cu', 'cuh']):
        print('Formatting "{}"'.format(fn))
        os.system('{} -i -style=file {}'.format(clang_format_bin, fn))
        return True
    elif has_suffix(fn, ['txt', 'md', 'rst', 'cfg', 'll', 'ptx']):
        print('Formatting "{}"'.format(fn))
        format_plain_text(fn)
        return True
    else:
        return False
```

Reply:
No, the \n formatter is (already) in format_plain_text, so we need to apply format_plain_text to .py and .cpp too.

Reply:
I mean, please use `returns` in all branches for more clarity. (Updated the suggested change.)